platform/default/sqlite3.hpp

```diff
@@ -85,8 +87,11 @@ class Statement {
     void reset();
     void clearBindings();
 
+    int64_t lastInsertRowId() const;
+    int64_t changes() const;
```

Comment:
We can, though my rationale for moving it back to a signed value is because the signature for [sqlite3_changes](https://www.sqlite.org/c3ref/changes.html) returns an `int`, which is also mirrored in [QSqlQuery::numRowsChanged](http://doc.qt.io/qt-5/qsqlquery.html#numRowsAffected) - the latter even uses `-1` to report an error state e.g. when the statement is not _active_. My concern is that mixing signed and unsigned return values is prone to integer overflow bugs.